test/jdk/java/nio/file/Files/SBC.java

```diff
@@ -63,6 +63,7 @@ public static void main(String[] args) throws Exception {
             badCombinations(dir);
             unsupportedOptions(dir);
             nullTests(dir);
+            emptyPathTest();
```

Comment:
I think the `@bug` of this test would also have to be updated to include `8349812`.

Reply:
> I think the `@bug` of this test would also have to be updated to include `8349812`.

Also, the copyright year update is missing in both files. I'll approve this once those three changes are made. Whether other tests for empty paths are needed can be evaluated separately.

Reply:
Addressed both concerns with the new commit, thanks!